### PR TITLE
Fix[SWA-160]: Added the correct counter for the number of submissions

### PR DESCRIPTION
**Pull request by Gon4ar5**
Resolve [SWA-160](https://artsyproduct.atlassian.net/browse/SWA-160)

Fix submission counter, cause he didn't correctly count the number of anonymous user submissions

### CODE REVIEW

File: app/models/submission.rb [hunk 1]
```diff
@@ -196,4 +196,10 @@ def name
   def phone
     user_phone || user&.phone
   end
+
+  def count_submissions_of_user
+    return Submission.where(user_email: user_email).count if user.nil?
```
Comment (Gon4ar5) [inline app/models/submission.rb:201]: What do you think about finding submissions exclusively by the same emails?

Comment (kiryl-zubarau) [reply app/models/submission.rb:201]: I suppose It should also ignore non anonymous submissions.

Comment (Gon4ar5) [reply app/models/submission.rb:201]: If our submission is anonymous, then a user is not created for it

Comment (Gon4ar5) [reply app/models/submission.rb]: Resolved

Comment (kiryl-zubarau) [reply app/models/submission.rb]: I mean that if the submission is anonymous you should search among other anonymous submissions instead of all of their

Comment (Gon4ar5) [reply app/models/submission.rb]: I understand, but now it doesn't matter anymore :)

**Review by iskounen — APPROVED** [review]

**Pull request merged**